utilities/virt.py

```diff
@@ -1579,16 +1579,22 @@ def get_rhel_os_dict(rhel_version):
     raise KeyError(f"Failed to extract {rhel_version} from system_rhel_os_matrix")
 
 
-def assert_vm_not_error_status(vm: VirtualMachineForTests) -> None:
+@retry(
+    wait_timeout=TIMEOUT_2MIN,
+    sleep=TIMEOUT_5SEC,
+    exceptions_dict={AssertionError: []},
+)
+def assert_vm_not_error_status(vm: VirtualMachineForTests) -> bool:
     status = vm.instance.get("status")
     printable_status = status.get("printableStatus")
     error_list = VM_ERROR_STATUSES.copy()
     vm_devices = vm.instance.spec.template.spec.domain.devices
     if vm_devices.gpus:
         error_list.remove(VirtualMachine.Status.ERROR_UNSCHEDULABLE)
-    assert printable_status not in error_list, (
-        f"VM {vm.name} error printable status: {printable_status}\nVM status:\n{status}"
-    )
+    if printable_status not in error_list:
+        return True
```

Comment:
are we going to use this return value? we can keep the retry logic but let it return none? wdyt